specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/preview/2018-06-30-preview/guestconfiguration.json

```diff
@@ -852,6 +1013,323 @@
         }
       },
       "description": "Reason and code for the compliance of the guest configuration assignment resource."
+    },
+    "SoftwareUpdateProfile": {
```

Comment:
in the properties, any property which can take a value from a fixed set should be modeled as an enum. Any bools should be modeled as string enum. And and dates, should have "format": "date-time"

Reply:
date-time reference added where possible, enums considered where possible

Reply:
Done where possible